toxics/latency.go

```diff
@@ -21,7 +21,7 @@ func (t *LatencyToxic) delay() time.Duration {
 	delay := t.Latency
 	jitter := t.Jitter
 	if jitter > 0 {
-		//#nosec
+		// #nosec G404 -- was ignored before too
```

Comment:
Adding the comment in the standard way as mentioned in the [documentation](https://github.com/securego/gosec?tab=readme-ov-file#annotating-code).